utils/scripts/openshift/openshift.py

```diff
@@ -88,6 +87,23 @@ def _aws_configure(self):
         configure_cmd.expect("Default output format .*: ")
         configure_cmd.sendline("yaml")
         configure_cmd.interact()
+        """
+
+        contents = "aws configure << EOF \n{}\n{}\n{}\n{}\nEOF".format(
+            self.aws_access_key_id,
+            self.aws_secret_access_key,
+            self.aws_region,
+            "yaml",
+        )
+
+        with open("aws.sh", "w") as f:
```

Comment:
from where are we going to pull aws. sh file?

Reply:
On the directory where u are running the script.